iib/web/models.py

```diff
@@ -30,6 +30,7 @@ class RequestStateMapping(BaseEnum):
     in_progress = 1
     complete = 2
     failed = 3
+    removed = 4
```

Comment:
A new state is not necessary. Remember, this state is about the request and not the index image. So `in_progress` => `complete` or `failed` works just fine with the remove workflow.